To: vim_dev@googlegroups.com
Subject: Patch 7.3.375
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.375
Problem:    Duplicate return statement.
Solution:   Remove the superfluous one. (Dominique Pelle)
Files:	    src/gui_mac.c


*** ../vim-7.3.374/src/gui_mac.c	2011-06-19 01:14:22.000000000 +0200
--- src/gui_mac.c	2011-12-10 17:20:59.000000000 +0100
***************
*** 3216,3222 ****
  {
      /* TODO:
       * This proc is called when Normal is set to a value
!      * so what msut be done? I don't know
       */
  }
  
--- 3216,3222 ----
  {
      /* TODO:
       * This proc is called when Normal is set to a value
!      * so what must be done? I don't know
       */
  }
  
***************
*** 3303,3309 ****
      *x = bounds.left;
      *y = bounds.top;
      return OK;
-     return FAIL;
  }
  
  /*
--- 3303,3308 ----
*** ../vim-7.3.374/src/version.c	2011-12-14 14:49:41.000000000 +0100
--- src/version.c	2011-12-14 15:06:01.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     375,
  /**/

-- 
GALAHAD: No look, really, this isn't nescess ...
PIGLET:  We must examine you.
GALAHAD: There's nothing wrong with ... that.
                 "Monty Python and the Holy Grail" PYTHON (MONTY) PICTURES LTD

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
